src/solver/conv_asm_implicit_gemm_v4r1_dynamic.cpp

```diff
@@ -278,6 +280,10 @@ bool ConvAsmImplicitGemmV4R1DynamicFwd::IsApplicable(const ConvolutionContext& c
         return false;
 
     const auto device_name = ctx.GetStream().GetDeviceName();
+#if WORKAROUND_ISSUE_1146
```

Comment:
Similarly, redundant conditions due to line 281/287?

Reply:
+1